array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+function unique(array) {
+  // create new array to keep track of oldElements
+  var oldElements = [];
+  // loop through array, checking if each element is in oldElements
+  for (var i = 0; i < array.length; i++) {
+    if (oldElements.includes(array[i])) {
+      continue;
```

Comment:
Instead of using `continue` here, you can use the `!` operator to invert your condition for this conditional. In general, we want to avoid having a code block whose only job is to `continue`, since that usually means that the code block is searching for something you don't actually need.